gigalixir/openers/windows.py

```diff
@@ -0,0 +1,8 @@
+from gigalixir.shell import cast
+import logging
+
+class WindowsOpener(object):
+    def open(self, url):
+        logging.getLogger("gigalixir-cli").info("Running: start %s" % url)
+        cast("start %s" % url)
```

Comment:
I'm not that familiar with windows. Does the start command open up a new browser when given a url?

Reply:
Yup. It will launch programs that you give it, but if you just give it a URL it will open in the default browser.